File: api/code-samples/python/klay/filter/getLogs.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

filterOptions = {
    "fromBlock": "latest",
    "toBlock": "latest",
    "address": "0x87ac99835e67168d4f9a40580f8f5c33550ba88b"
}

sdk = OpenSDK(host)
klay_response = sdk.klay.get_logs(filterOptions)

print(json.loads(klay_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/klay/filter/getLogs.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

  const filterOptions = {
    fromBlock: "latest",
    toBlock: "latest",
    address: "0x87ac99835e67168d4f9a40580f8f5c33550ba88b",
  };

  provider.klay
    .getLogs(filterOptions, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
